tests/solids/diamondC_1x1x1_pp/det_qmc_input_check_broken_xml.in.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<simulation>
+   <project id="det_qmc_input_check_broken_xml" series="0">
```

Comment:
It might be useful to have comments in these files with a little more explanation of what's wrong with the input file.  (For this one, how is the XML broken?)

Reply:
I'll add some comments. This one is heavily truncated.

Reply:
Done